lib/Entry.hs

```diff
@@ -76,11 +78,18 @@ instance A.FromJSON FieldVisibility where
     "private" -> pure Private
     other -> fail $ "expecting either 'public' or 'private', but found: '" <> T.unpack other <> "'"
 
+newtype FieldValue = FieldValue { fvValue :: T.Text }
```

Comment:
We should now be using this type everywhere (e.g. see `CLI.Types`, possibly other modules too).

Reply:
Done (I think)

Reply:
See also:
* `newEntry` should take a `FieldValue`
* `parseFieldContentsEof` should return a `FieldValue`
*

Reply:
Don't get where `newEntry` should take a `FieldValue`. At this moment its signature is
````haskell
newEntry :: EntryPath -> UTCTime -> Entry
````

Reply:
Oops I meant `newField` :sweat_smile:

Reply:
Done